src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_KEY = "supply";
+    private static final String BUY_KEY = "buy";
+    private static final String RESULT_KEY = "result";
+    private static final int SUPPLY_INDEX = 0;
+    private static final int BUY_INDEX = 1;
+    private static final String COMMA = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        try (
+                BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+                BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName, true))
+            ) {
+            int[] data = getDataFromFile(reader);
+            writeDataToFile(writer, data);
+
+        } catch (IOException e) {
+            throw new RuntimeException("I can't read file", e);
```

Comment:
which file? and why read? let's create separate try-with-resources for reader and writer